webpack/scenes/ContentViews/Table/tableDataGenerator.js

```diff
@@ -27,7 +27,7 @@ const buildRow = (contentView, openColumn) => {
     id, composite, name, environments, versions, last_published: lastPublished,
   } = contentView;
   const row = [
-    { title: <ContentViewIcon composite={composite ? true : undefined} /> },
+    { title: <ContentViewIcon composite={composite ? true : undefined} />, props: { id, name } },
```

Comment:
@sjha4 is this still needed? Looks like the info is coming from the row data that is set up below.

Reply:
Right..This is not needed..